modules/apps/site/site-cms-site-initializer/src/main/resources/META-INF/resources/js/main_view/props_transformer/utils/displayUndoDeleteSuccessToast.ts

```diff
@@ -10,6 +10,7 @@ import restoreItemAction from '../actions/restoreItemAction';
 
 const recycleBinToastInfo = {
 	className: 'recycle-bin-link',
+	componentId: 'recycle-bin-successToastWithLink',
```

Comment:
Hi @rebsilva, I did a search on the portal and noticed that there are examples for the approach followed in this PR, but I couldn't find anyone naming the componentId in that format, did you base it on any example?